examples/unit_conversion.ipynb

```diff
@@ -0,0 +1,1265 @@
+       " 1.07763,\n",
+       " 1.0786300000000002,\n",
+       " 1.0796400000000002,\n",
+       " 1.08064,\n",
+       " 1.0816400000000002,\n",
+       " 1.0826500000000001,\n",
+       " 1.0836500000000002,\n",
+       " 1.0846600000000002,\n",
+       " 1.08566,\n",
+       " 1.08667,\n",
+       " 1.0876700000000001,\n",
+       " 1.08868,\n",
+       " 1.08969,\n",
+       " 1.0907,\n",
+       " 1.09172,\n",
+       " 1.09273,\n",
+       " 1.09374,\n",
+       " 1.09476,\n",
+       " 1.09577,\n",
+       " 1.09678,\n",
+       " 1.0977999999999999,\n",
+       " 1.09882,\n",
+       " 1.09985,\n",
+       " 1.10087,\n",
+       " 1.10189,\n",
+       " 1.10291,\n",
+       " 1.10393,\n",
+       " 1.10494,\n",
+       " 1.10596,\n",
+       " 1.10698,\n",
+       " 1.108,\n",
+       " 1.10901,\n",
+       " 1.11003,\n",
+       " 1.1110499999999999,\n",
+       " 1.11207,\n",
+       " 1.1130799999999998,\n",
+       " 1.11409,\n",
+       " 1.1151099999999998,\n",
+       " 1.11612,\n",
+       " 1.11714,\n",
+       " 1.11816,\n",
+       " 1.11917,\n",
+       " 1.12019,\n",
+       " 1.1212,\n",
+       " 1.1222100000000002,\n",
+       " 1.12321,\n",
+       " 1.12422,\n",
+       " 1.12523,\n",
+       " 1.1262400000000001,\n",
+       " 1.12726,\n",
+       " 1.12828,\n",
+       " 1.1293,\n",
+       " 1.13032,\n",
+       " 1.13133,\n",
+       " 1.13235,\n",
+       " 1.13337,\n",
+       " 1.1343900000000002,\n",
+       " 1.13541,\n",
+       " 1.13643,\n",
+       " 1.13745,\n",
+       " 1.13846,\n",
+       " 1.13948,\n",
+       " 1.1405,\n",
+       " 1.14152,\n",
+       " 1.14254,\n",
+       " 1.14356,\n",
+       " 1.14458,\n",
+       " 1.1456,\n",
+       " 1.14663,\n",
+       " 1.1476600000000001,\n",
+       " 1.14869,\n",
+       " 1.14972,\n",
+       " 1.15074,\n",
+       " 1.1517600000000001,\n",
+       " 1.15279,\n",
+       " 1.15382,\n",
+       " 1.15485,\n",
+       " 1.1558800000000002,\n",
+       " 1.15691,\n",
+       " 1.1579300000000001,\n",
+       " 1.15896,\n",
+       " 1.15999,\n",
+       " 1.16102,\n",
+       " 1.16205,\n",
+       " 1.16308,\n",
+       " 1.1641,\n",
+       " 1.1651300000000002,\n",
+       " 1.16616,\n",
+       " 1.16718,\n",
+       " 1.1682000000000001,\n",
+       " 1.1692200000000001,\n",
+       " 1.17025,\n",
+       " 1.17127,\n",
+       " 1.17231,\n",
+       " 1.17334,\n",
+       " 1.1743800000000002,\n",
+       " 1.17541,\n",
+       " 1.17645,\n",
+       " 1.17748,\n",
+       " 1.17851,\n",
+       " 1.1795499999999999,\n",
+       " 1.18059,\n",
+       " 1.18162,\n",
+       " 1.18266,\n",
+       " 1.1837,\n",
+       " 1.1847400000000001,\n",
+       " 1.18578,\n",
+       " 1.18682,\n",
+       " 1.18787,\n",
+       " ...]"
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "uuid = \"8d6b034e-efb4-427a-a0d3-666e62762cba\"\n",
```

Comment:
I'm not sure if it's a good idea to assume this kitem exists. This part failed for me when running it on my local instance. Maybe check first if it exists and if not, create it?

Reply:
Notebook has been deleted.